torch/_inductor/compile_fx.py

```diff
@@ -936,15 +936,17 @@ def clone_preserve_strides(x: torch.Tensor):
 
 
 def copy_misaligned_inputs(
-    new_inputs: List[torch.Tensor], check_inputs_idxs: Sequence[int]
+    new_inputs: List[Union[torch.Tensor, int]], check_inputs_idxs: Sequence[int]
```

Comment:
Can you factor out the `Union[torch.Tensor, int]` alias into torch/_inductor/utils, and then reuse it ? this will make it easier to add new types if it comes up.